src/odemis/util/transform.py

```diff
@@ -738,6 +739,15 @@ def __init__(
             numpy.zeros(2, dtype=float) if translation is None else translation
         )
 
+    def __matmul__(self, other):
+        """Overloads the `@` operator to combine two transforms into one."""
```

Comment:
It'd be nice to clarify the order in the documentation. Something like:
"(A @ B).apply(p) is equivalent to A.apply(B.apply(p))"